application-development/helloworld-springboot-auth/.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
remove this file?

I don't think we should be committing any IDE configuration files.
thanks